.github/workflows/node.js.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Run Google es-linter
+      run: npm run lint
```

Comment:
you can use the google js styleguide from here: https://github.com/google/styleguide

I got the pylint from here: https://github.com/google/styleguide/blob/ea180f7259da4ae90ae086c1eefbeb5f440dbea2/pylintrc

Reply:
yeah, trying to use "eslint-config-google": "^0.14.0", from ibr package but there's some errors